code/modules/mob/living/carbon/human/human.dm

```diff
@@ -788,7 +788,7 @@
 /mob/living/carbon/human/check_weakness(obj/item/weapon, mob/living/attacker)
 	. = ..()
 	if (dna && dna.species)
-		. += dna.species.check_weakness(weapon, attacker)
+		. += dna.species.check_species_weakness(weapon, attacker)

```

Comment:
Nich llease replacw this with a multiply and change the other procs accordingly because using add for this is FUCKING RETARDED

Reply:
This is correct actually. 

https://github.com/tgstation/tgstation/blob/732b94093525282c2e3dd49f9d864995913a2cf3/code/modules/mob/living/carbon/human/species.dm#L1338-L1339

https://github.com/tgstation/tgstation/blob/f04d6fc8d1bbaa951fa89e9e531a6d256350bff7/code/modules/antagonists/devil/true_devil/_true_devil.dm#L122-L123

Reply:
Right or not, it's dumb :)